src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseStudyServiceImpl.java

```diff
@@ -281,6 +296,9 @@ public T create(IUser owner, T study) throws NotUniqueException, NotExistExcepti
         study.setType(studyTypeService.getById(study.getType().getId()));
         study.setStatus(studyStatusService.findByName("Initiate"));
         study.setTenant(owner.getActiveTenant());
+        if (study.getKind() == null) {
```

Comment:
it would be better to control default value in one place, for example exactly in the entity